dp-icea/dss
# Logger service implementation

- Added fluentd writer to DSS log pkg (this allows the API to send logs directly to TCP endpoint provided by Fluent);
- Added kong plugins for log forwarding and processing globally;
- Implemented fluentd configuration for receiving and parsing logs from DSS and Kong gateway APIs;
- Added makefile command for opensearch container build.

How to test:
`make opensearch` 
`make kong`
`bash ./build/dev/run_locally.sh up -d`

Make requests do DSS and Kong gateway and check the logs for fluentd container, they should appear.
To validate **opensearch** operation, you can access the dashboard at: http://localhost:5601/app/opensearch-query-
- A logstash file with current date will be created after fluentd receives logs

- You can also check http://localhost:5601/app/observability-logs#/explorer with timestamp filter value for the current day.
